src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj

```diff
@@ -4,8 +4,7 @@
 
   <PropertyGroup>
     <Description>NuGet's understanding of target frameworks.</Description>
-    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">$(TargetFrameworksLibrary);netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworksLibrary);net40</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">$(TargetFrameworksLibrary);net40</TargetFrameworks>
```

Comment:
The probably is TargetFrameworks is never set in this project.

Reply:
Fixed, thanks!